.github/workflows/main.yml

```diff
@@ -4,33 +4,50 @@ jobs:
   install-and-test:
     runs-on: macos-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-java@v3
+
+      - name: Cleanup
+        run: |
+          sudo rm -rf /Users/runner/.gradle/caches
+          sudo rm -rf /Library/Developer/CoreSimulator/Profiles/Runtimes/*
+          sudo rm -rf ~/Library/Developer/Xcode/DerivedData/*
+          sudo rm -rf ~/Library/Developer/Xcode/Archives/*
+          sudo rm -rf ~/Library/Caches/*
+          sudo rm -rf /Users/runner/Library/Developer/CoreSimulator/Caches/*
+          sudo rm -rf /Users/runner/Library/Developer/CoreSimulator/Devices/*
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
         with:
           java-version: '17'
           distribution: 'temurin'
-      - uses: subosito/flutter-action@v1
+
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v1
         with:
-          channel: 'stable' # or: 'beta', 'dev' or 'master'
-      - run: flutter pub get
-      # Android
-      - run: |
+          channel: 'stable'
+      
+      - name: Install dependencies
+        run: flutter pub get
+
+      - name: Android
+        run: |
           cd example
           flutter pub get
           cd android
           ./gradlew clean build -PSQUARE_READER_SDK_APPLICATION_ID=${{secrets.SQUARE_READER_SDK_APPLICATION_ID}} -PSQUARE_READER_SDK_REPOSITORY_PASSWORD=${{secrets.SQUARE_READER_SDK_REPOSITORY_PASSWORD}}
-      # #IOS
-      - run: |
+
+      - name: iOS
+        run: |
           cd example
           flutter pub get
           cd ios
-          ruby <(curl https://connect.squareup.com/readersdk-installer) install --app-id ${{secrets.SQUARE_READER_SDK_APPLICATION_ID}} --repo-password ${{secrets.SQUARE_READER_SDK_REPOSITORY_PASSWORD}} --version 1.6.1 > /dev/null
+          ruby <(curl https://connect.squareup.com/readersdk-installer) install --app-id ${{secrets.SQUARE_READER_SDK_APPLICATION_ID}} --repo-password ${{secrets.SQUARE_READER_SDK_REPOSITORY_PASSWORD}} --version 1.7.4 > /dev/null
```

Comment:
is it possible to define the version as a variable somewhere, similar to ${{secrets.SQUARE_READER_SDK_APPLICATION_ID}} ?

Reply:
I think you create an env yaml file and use it like this https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#using-the-env-context-to-access-environment-variable-values I'd much rather go that way because the app version isn't technically a secret